test/query_test.js

```diff
@@ -519,12 +519,17 @@ describe('query', function () {
     return Promise.all([p1, p2]);
   });
 
+  it('ref', function() {
+    var id = refN1.id.toString();
+    return assertQuery(query.Ref(classRef, query.Concat([id.charAt(0), id.substr(1)])), refN1);
```

Comment:
I didn't understand this assertion. Can't we just use literals to make it more clear?